### PR TITLE
Merge gfx940 to staging 6.0

### PR DESCRIPTION
**Pull request by cgmillette**

### CODE REVIEW

**Review by mkarunan — APPROVED** [review]

**Review by dlangbe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 hiptensor is AMD's C++ library for accelerating tensor primitives based on the composable kernel library, through general purpose kernel languages, like HIP C++.
 
 ## GPU Support
-* AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a as 'gfx9'
+* AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940, gfx941, gfx942 as 'gfx9'
```
Comment (Rmalavally) [inline README.md:5]: We could consider adding a link to the GPU Support matrix and ensure that this information is consistent with the matrix.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-`Note: Double precision FP64 datatype support requires gfx90a`
+`Note: Double precision FP64 datatype support requires gfx90a, gfx940, gfx941 or gfx942`
 
 ## Minimum Software Requirements
 * ROCm stack minimum version 5.7
```
Comment (Rmalavally) [inline README.md:10]: Any impact for 5.7.1?

File: docs/Linux_Install_Guide.rst [hunk 3]
```diff
@@ -38,9 +38,9 @@ As a general rule, 8GB of system memory is required for a full hipTensor build.
 
 GPU Support
 ^^^^^^^^^^^
-AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a as 'gfx9'
+AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940, gfx941, gfx942 as 'gfx9'
```
Comment (Rmalavally) [inline docs/Linux_Install_Guide.rst:41]: Please ensure consistency with what's published in the GPU Support matrix on the portal.

File: README.md (hunk 2 above)
Comment (mkarunan) [reply README.md:10]: With double precision types  ?

**Pull request merged**